BUILDING.md

```diff
@@ -60,12 +60,23 @@ Node.js relies on V8 and libuv. We adopt a subset of their supported platforms.
 There are three support tiers:
 
 * **Tier 1**: Full test coverage and maintenance by the Node.js core team and
-  the broader community.
+  the broader community. All commits to the Node.js repository are tested on
+  multiple variants of these platforms. Breakages are considered critical for
+  the purpose of creating releases.
```

Comment:
This sentence seems to boil down to "Breakages are critical to creating releases." That can't be right, but I'm not sure what it's supposed to mean. Is it supposed to indicate that failing tests on Tier 1 platforms will block a release? Or something else?